src/ecs/resources/input.rs

```diff
@@ -62,45 +80,63 @@ impl<'a> Iterator for PressedMouseButtons<'a> {
 /// ```
 #[derive(Default)]
 pub struct InputHandler {
-    pressed_keys: HashMap<VirtualKeyCode, KeyQueryState>,
-    pressed_mouse_buttons: HashMap<MouseButton, KeyQueryState>,
+    pressed_keys: Vec<VirtualKeyCode>,
```

Comment:
It would be great if we could instead just have a boolean `Vec` where the index is the key. Also see [this issue](https://github.com/amethyst/amethyst/issues/225).